src/odemis/gui/cont/tabs.py

```diff
@@ -4235,6 +4235,16 @@ def Show(self, show=True):
             # Cancel autofocus (if it happens to run)
             self.tab_data_model.autofocus_active.value = False
 
+            # Cancel manual focus: just reset the button, as the rest is just
```

Comment:
What about the slit of the spectrograph? For manual focus we also automatically close it. Or is this always taken care of by the optical path settings?

Reply:
Yes, the optical path manager takes care of the slit.